examples/custom_model/main.go

```diff
@@ -109,12 +110,14 @@ func (p *ExamplePlugin) AfterInit() (err error) {
 	go func() {
 		time.Sleep(time.Second)
 
-		//c := remoteclient.NewClientGRPC(api.NewGenericManagerClient(conn))
-		c := client.LocalClient
-
+		// remoteclient
+		c := remoteclient.NewClientGRPC(genericmanager.NewGenericManagerClient(p.conn))
 		demonstrateClient(c)
 
-		time.Sleep(time.Second * 3)
+		//time.Sleep(time.Second * 3)
+
+		// localclient
+		//demonstrateClient(client.LocalClient)
```

Comment:
You could remove all this clutter.

Reply:
Not a clutter. This is actually how the example will work after some upcoming changes. 

This example will most likely ask user to pick client implementation used for configuring agent. This should demonstrate that the client API can be kept exactly same for all client implementations (local, remote grpc, remote kvdb...)

Reply:
Then I would comment it with a little explanation so it does not look like forgotten commented code, for example:
````
// choose client implementation (remote, local, ...)

// remoteclient
c := remoteclient.NewClientGRPC(genericmanager.NewGenericManagerClient(p.conn))
demonstrateClient(c)

// localclient
//demonstrateClient(client.LocalClient)	

...
````
plus `time.Sleep()` where needed

Reply:
The localclient is not fully supported. I will un-comment this and ask for user input to pick implementation when all are ready and fully implemented.